code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -83,8 +83,9 @@
 		CHECK_TICK
 
 /proc/purrbation_toggle(mob/living/carbon/human/H, silent = FALSE)
-	if(!ishumanbasic(H))
-		return
+	if(!ishumanbasic(H) && !iscatperson(H))
+		purrbation_apply_mutant(H, silent)
+		return TRUE
```

Comment:
My main complaint is that it looks like this doesn't really make any faithful use of the `silent` parameter, in the non-human case.

Reply:
addressed